utils/http_utils.go

```diff
@@ -22,52 +23,59 @@ type HttpResponse struct {
 	StatusCode int
 }
 
-// It send an http post to the endpoint signing the body with the worker's private key
-func SignedPost(workerId string, body interface{}, endpoint string) *HttpResponse{
-	requestBody, err := json.Marshal(body)
+func getSignature(payload interface{}, workerId string) []byte {
+	parsedPayload, err := json.Marshal(payload)
 
 	if err != nil {
-		log.Fatal("Error on marshalling the request body")
+		log.Fatal("Error on marshalling the payload")
 	}
 
-	data, hashSum := SignMessage(GetPrivateKey(workerId), requestBody)
+	signedPayload, _ := SignMessage(GetPrivateKey(workerId), parsedPayload)
```

Comment:
I believe that `signature` is a better name for the` signedPayload` variable. What do you think?